doc-content/enterprise-only/getting-started/new-project-ds-proc.adoc

```diff
@@ -1,17 +1,19 @@
 [id='decision-services-project-create-proc']
 = Creating the traffic violations project
-A project is the container for assets such as data objects, guided decision tables, and guided rules. For this tutorial, you will create the *Driver_department_traffic_violations* project.
+A project is a container for assets such as data objects, guided decision tables, and guided rules.
+
 
 .Procedure
 . Log in to {CENTRAL}.
 . Go to *Menu* -> *Design* -> *Projects*.
 +
 
-*Tip*: Red Hat Decision Manager provides a default space called *MySpace*, as shown in the following image. You can use the default space to create and test example projects.
+*Tip*: {PRODUCT} provides a default space called *MySpace*, as shown in the following image. You can use the default space to create and test example projects.
 +
 
-.Default spaces
+.Defaul space
```

Comment:
This looks like a typo. It should probably be `.Default space`.

Reply:
A total typo! fixed.